src/webpackWorker.js

```diff
@@ -139,7 +139,7 @@ module.exports = function(configuratorFileName, options, index, expectedConfigLe
         }
         var compiler = webpack(webpackConfig);
         if(watch || webpack.watch) {
-            watcher = compiler.watch({}, finishedCallback);
+            watcher = compiler.watch(webpackConfig.watchOptions, finishedCallback);
```

Comment:
Just wondering what will happen if there are no `watchOptions`? Perhaps `webpackConfig.watchOptions || {}` would be safer?